docs/UserGuide.adoc

```diff
@@ -530,29 +564,44 @@ in the offline copy of bookmark 1. Highlight
 colour remains the same. Otherwise, paragraph 2 will be highlighted yellow and
 a new note with content "change or add note content" will be added to the paragraph.
 
-* `*annotate* 1 n/adding a general note` +
+* `*annotate* 1 p/null n/adding a general note` +
 This adds a note with content "adding a general note" to the _General notes section_ in the
 offline copy of bookmark 1.
 
+
 [[annotatedelete]]
 === Deleting annotations on an offline copy: *`annotate-delete`*
 
-This deletes selected highlights or notes from the offline copy of the given bookmarked website.
-When you give this command, the UI will switch to the offline tab showing the results of your command.
+If you want to delete highlights or notes from the offline copy of a bookmark, you can do so using the `annotate-delete` command.
 You can choose to remove just the notes and/or highlight of a paragraph, or clear all annotations
 on the offline copy to revert it to a clean slate. You can also choose to remove a note from the
-<<stray-notes,_General notes section_>>. If the given paragraph does not have any annotations to remove, nothing is performed.
+<<stray-notes,_General notes section_>>.
```

Comment:
Just a note: After I merge the reorganised User Guide, you can put the link to "General notes section" at the start of the [Offline Copies](https://deploy-preview-248--mark-cs2103t.netlify.com/userguide#offline-copies) section.